### DISCUSSION TITLE
How to implement tsdx

### DISCUSSION THREAD

**Discussion by MahdiTa97** [vercel/turborepo#690] [Help] [answered]
Please suggest a way to implement tsdx inside of Turborepo.
I implemented it but I can't run the watch script in the tsdx package.
this is my personal repo: https://github.com/MahdiTa97/turborepo-boilerplate
the tsdx package is: https://github.com/MahdiTa97/turborepo-boilerplate/tree/main/packages/my-theme-kit

![image](https://user-images.githubusercontent.com/20884619/152807846-3600dcfa-cc0d-45ea-aefc-f4279911244e.png)

as you can see, `theme-kit.cjs.development.js` doesn't exist in my dist folder!

**Comment by anthonyshew** [comment, marked as answer]
This was likely a misconfiguration of packaging. Marking as outdated!
